cmd/katalyst-agent/app/options/sysadvisor/qosaware/resource/resource_advisor.go

```diff
@@ -47,14 +47,16 @@ func (o *ResourceAdvisorOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&o.ResourceAdvisors, "resource-advisors", o.ResourceAdvisors, "active dimensions for resource advisors")
 
 	o.CPUAdvisorOptions.AddFlags(fs)
+	o.MemoryAdvisorOptions.AddFlags(fs)
 }
 
 // ApplyTo fills up config with options
 func (o *ResourceAdvisorOptions) ApplyTo(c *resource.ResourceAdvisorConfiguration) error {
 	c.ResourceAdvisors = o.ResourceAdvisors
 
 	var errList []error
-	errList = append(errList, o.CPUAdvisorOptions.ApplyTo(c.CPUAdvisorConfiguration))
+	errList = append(errList, o.CPUAdvisorOptions.ApplyTo(c.CPUAdvisorConfiguration),
+		o.MemoryAdvisorOptions.ApplyTo(c.MemoryAdvisorConfiguration))
```

Comment:
Follow the ApplyTo style as other components' way. e.g.
errList = append(errList, xxx.ApplyTo(xxx))
errList = append(errList, xxx.ApplyTo(xxx))
...

Reply:
done